AbletonMCP_Remote_Script/handlers/devices.py

```
@@ -0,0 +1,1381 @@
+        raise
+
+
+# --- Simpler-to-Drum-Rack Conversion ---
+
+
+def sliced_simpler_to_drum_rack(song, track_index, device_index, ctrl=None):
+    """Convert a sliced Simpler device to a Drum Rack."""
+    try:
+        track = get_track(song, track_index)
+        if device_index < 0 or device_index >= len(track.devices):
+            raise IndexError("Device index out of range")
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)